interactives/action-menu/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <!--Let browser know website is optimized for mobile-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <!-- ### Enter your interactive title in the title block ### -->
+    <title>Action Menu</title>
+    <link rel="stylesheet" href="../base-files/css/third-party/normalize.css">
+    <!--Import materialize.css-->
+    <link rel="stylesheet" href="../base-files/css/third-party/materialize.min.css" type="text/css" media="screen,projection"/>
+    <script src="../base-files/js/third-party/modernizr.js"></script>
+    <!-- jQuery (necessary for Materialize) -->
+    <script type="text/javascript" src="../base-files/js/third-party/jquery.js"></script>
+  </head>
+  <body>
+    <div class="row interactive center-align">
+    <div class="s12 col">
+    <!-- ### Start of required HTML for inline interactives ### -->
+      <a id='dropdownMenu' class='dropdown-button btn' href='#' data-activates='dropdown-available-menu-items' data-beloworigin='true'>Available Menu Items</a>
```

Comment:
It would be good for both interactives to use more precise selectors, that won't interfere with other interactives. You don't need to worry about this if the interactive is being used in `whole-page` mode (which I believe the `high-score-boxes` is designed for this), however `in-page` interactives should be more precisely specified. I can show you this when I see you next.